crates/tools/sys/src/main.rs

```diff
@@ -3,7 +3,35 @@ use std::collections::*;
 use std::io::prelude::*;
 
 /// Namespaces to exclude from code generation for the `windows-sys` crate.
-const EXCLUDE_NAMESPACES: [&str; 4] = ["Windows.Win32.System.WinRT.Xaml", "Windows.Win32.Interop", "Windows.Win32.System.Diagnostics.Debug.WebApp", "Windows.Win32.Web"];
+const EXCLUDE_NAMESPACES: [&str; 27] = [
+    "Windows.Win32.AI.MachineLearning",
+    "Windows.Win32.Graphics.CompositionSwapchain",
+    "Windows.Win32.Graphics.Direct2D",
+    "Windows.Win32.Graphics.Direct3D",
+    "Windows.Win32.Graphics.Direct3D10",
+    "Windows.Win32.Graphics.Direct3D11",
+    "Windows.Win32.Graphics.Direct3D11on12",
+    "Windows.Win32.Graphics.Direct3D12",
+    "Windows.Win32.Graphics.Direct3D9",
+    "Windows.Win32.Graphics.Direct3D9on12",
+    "Windows.Win32.Graphics.DirectComposition",
+    "Windows.Win32.Graphics.DirectDraw",
```

Comment:
Wouldn't things like wgpu want those in windows-sys? (cc @Elabajaba)

Reply:
I don't think so? As far as I know `wgpu` needs COM support (and like Kenny said, these are all probably unusable without COM), which means `wgpu` needs to use `windows` instead of `windows-sys` anyways.